src/wp-includes/pluggable.php

```diff
@@ -2180,34 +2180,27 @@ function wp_hash($data, $scheme = 'auth') {
 }
 endif;
 
-if ( !function_exists('wp_hash_password') ) :
+if ( !function_exists( 'wp_hash_password' ) ) :
 /**
- * Create a hash (encrypt) of a plain text password.
+ * Create a hash (encrypt) of a plain text password using bcrypt.
+ * 
+ * See http://php.net/manual/en/function.password-hash.php
  *
  * For integration with other applications, this function can be overwritten to
- * instead use the other package password checking algorithm.
+ * instead use a different password checking algorithm.
  *
- * @since WP-2.5.0
- *
- * @global PasswordHash $wp_hasher PHPass object
+ * @since CP-1.0.0
  *
  * @param string $password Plain text user password to hash
  * @return string The hash string of the password
  */
-function wp_hash_password($password) {
-	global $wp_hasher;
-
-	if ( empty($wp_hasher) ) {
-		require_once( ABSPATH . WPINC . '/class-phpass.php');
-		// By default, use the portable hash from phpass
-		$wp_hasher = new PasswordHash(8, true);
-	}
-
-	return $wp_hasher->HashPassword( trim( $password ) );
+function wp_hash_password( $password ) {
+	$options = apply_filters( 'wp_hash_password_options', [] );
```

Comment:
New filter needs inline documentation.  And since it's a new one, may as well change it to `cp_` too.